src/foam/nanos/auth/PermissionedPropertyDAO.js

```diff
@@ -56,8 +56,13 @@ foam.CLASS({
       javaCode: `
       if ( x.get("auth") != null ) {
         if ( predicate != null ) predicate.authorize(x);
-        foam.dao.Sink sink2 = ( sink != null ) ? new HidePropertiesSink(x, sink, this) : sink;
-        super.select_(x, sink2, skip, limit, order, predicate);
+          // don't decorate the sink if it's a Count or GroupBy
+          if ( ! ( sink instanceof foam.mlang.sink.Count || sink instanceof foam.mlang.sink.GroupBy) ) {
```

Comment:
Only skip if its a GroupBy(Count()), if it were a GroupBy() of something else, then we still decorate.